Automattic/jetpack
# Sync Checksums : limit users based on user_level

Reduce included users in checksum to contributor and above users. This is performed by an explicit check of the {$wpdb->prefix}user_level meta which correlates to a non 0 value.

Fixes https://wp.me/p9dueE-3JV

#### Changes proposed in this Pull Request:
* Limit the scope of User and User Meta checksums to contributor and above users.

#### Does this pull request change what data or activity we track or use?
NO

#### Testing instructions:

Pre-Applying Branch
* Enter WP Shell
* $store = new Automattic\Jetpack\Sync\Replicastore();
* $checksum_table = $store->get_table_checksum_instance( 'users', '', false );
* $checksum_table->get_range_edges();
* // Verify item_count matches number of users on your site
* $checksum_table->calculate_checksum( null, null, null, true );
* // Store this value to compare checksums after applying branch.
* $checksum_table = $store->get_table_checksum_instance( 'usermeta', '', false );
* $checksum_table->get_range_edges();
* // Verify item_count matches number of users on your site
* $checksum_table->calculate_checksum( null, null, null, true );
* // Store this value to compare checksums after applying branch.

Apply Branch
* Enter WP Shell
* $store = new Automattic\Jetpack\Sync\Replicastore();
* $checksum_table = $store->get_table_checksum_instance( 'users', '', false );
* $checksum_table->get_range_edges();
* // Verify item_count matches number of users on your site that have contributor or above caps.
* $checksum_table->calculate_checksum( null, null, null, true );
* // Verify that checksum values match what was generated pre patch
* $checksum_table = $store->get_table_checksum_instance( 'usermeta', '', false );
* $checksum_table->get_range_edges();
* // Verify item_count matches number of users on your site that have contributor or above caps.
* $checksum_table->calculate_checksum( null, null, null, true );
* // Verify that checksum values match what was generated pre patch
